static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -75,17 +78,36 @@ CStudioAuthoring.Module.requireModule(
             }
           }
 
-          CStudioForms.Controls.RTEManager.getRteConfiguration(
-            configuration,
-            'no-role-support',
-            {
-              success: function (rteConfig) {
-                _thisControl._initializeRte(config, rteConfig, containerEl);
-              },
-              failure: function () {}
-            },
-            '/form-control-config/rte/rte-config.xml'
-          );
+          if (!Boolean(CrafterCMSNext.system.store?.getState().preview.richTextEditor)) {
+            // If textEditorConfig is not loaded
+            CrafterCMSNext.system.store.dispatch({
+              type: 'INIT_RICH_TEXT_EDITOR_CONFIG',
+              payload: {
+                configXml: CrafterCMSNext.system.store.getState().uiConfig.xml,
+                siteId: CStudioAuthoringContext.site
+              }
+            });
+            CrafterCMSNext.system
+              .getStore()
+              .pipe(
+                filter((store) => Boolean(store.getState().preview.richTextEditor)),
+                take(1)
+              )
```

Comment:
`getStore` only emits once when the store is created; if this call misses the RTE config initialization it will never go through; you need to subscribe to the store itself.